From: Johannes Weiner <hannes@cmpxchg.org>
Subject: mm-compaction-refactor-__compaction_suitable-fix
Date: Fri, 2 Jun 2023 10:49:42 -0400

Vlastimil points out the function comment is stale now. Update it.

Link: https://lkml.kernel.org/r/20230602144942.GC161817@cmpxchg.org
Signed-off-by: Johannes Weiner <hannes@cmpxchg.org>
Cc: Baolin Wang <baolin.wang@linux.alibaba.com>
Cc: Mel Gorman <mgorman@techsingularity.net>
Cc: Michal Hocko <mhocko@suse.com>
Cc: Vlastimil Babka <vbabka@suse.cz>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/compaction.c |    1 -
 1 file changed, 1 deletion(-)

--- a/mm/compaction.c~mm-compaction-refactor-__compaction_suitable-fix
+++ a/mm/compaction.c
@@ -2226,7 +2226,6 @@ static enum compact_result __compaction_
  * compaction_suitable: Is this suitable to run compaction on this zone now?
  * Returns
  *   COMPACT_SKIPPED  - If there are too few free pages for compaction
- *   COMPACT_SUCCESS  - If the allocation would succeed without compaction
  *   COMPACT_CONTINUE - If compaction should run now
  */
 enum compact_result compaction_suitable(struct zone *zone, int order,
_
